src/Components/AddToPlaylistModal.jsx

```diff
@@ -0,0 +1,79 @@
+import { usePlaylist } from "../Context";
+
+export function AddToPlaylistModal() {
+  const {
+    myPlaylists,
+    setShowModal,
+    newPlaylist,
+    setNewPlaylist,
+    createNewPlaylist,
+    targetVideo,
+    addVideoToPlaylist,
+    deleteVideoFromPlaylist,
+  } = usePlaylist();
+
+  const addingToPlaylist = (e) => {
+    if (e.key === "Enter") {
+      createNewPlaylist(newPlaylist);
+      setNewPlaylist("");
+    }
+  };
+
+  const togglePlaylist = (e, targetVideo) => {
+    if (e.target.checked) {
+      addVideoToPlaylist(e.target.id, targetVideo);
+    }
+    if (!e.target.checked) {
+      deleteVideoFromPlaylist(e.target.id, targetVideo._id);
+    }
+  };
+
+  const isInPlaylist = (targetVideo, playlistId) => {
+    const thisPlaylist = myPlaylists.find((item) => item._id === playlistId);
+    if (thisPlaylist?.videos.some((item) => item._id === targetVideo._id)) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  return (
+    <div className="add-to-playlist">
+      <h6>Add to Playlist</h6>
+      <img
+        src="../../assets/icons/close.svg"
+        id="close"
+        alt="close"
+        onClick={() => setShowModal(false)}
+      />
```

Comment:
close icon is not visible, kindly check it once

Reply:
yes I added the icon